electron/legendary/games.ts

```diff
@@ -302,7 +302,7 @@ class LegendaryGame extends Game {
     const withDlcs = installDlcs ? '--with-dlcs' : '--skip-dlcs'
     const installSdl = sdlList.length ? this.getSdlList(sdlList) : '--skip-sdl'
 
-    const logPath = `${join(heroicGamesConfigPath, this.appName + '.log')}`
+    const logPath = `"${join(heroicGamesConfigPath, this.appName + '.log')}"`
```

Comment:
Interesting how the quotes are required here. I would've thought that a Node-internal function (like `appendFile`) wouldn't need those

Reply:
Ah, you removed the quotes in a later commit
Whoops

Reply:
Yes, I noticed that we just need them when reading the path, so I removed them when declaring.